packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePlugin.kt

```diff
@@ -672,4 +655,78 @@ class AmplifyDataStorePlugin : FlutterPlugin, MethodCallHandler {
             }
         }
     }
+
+    private fun createErrorHandler(request: Map<String, Any>) : DataStoreErrorHandler{
+        var errorHandler : DataStoreErrorHandler
+        errorHandler = if( (request["hasErrorHandler"] as? Boolean? == true) ) {
+            DataStoreErrorHandler {
+                val args = hashMapOf(
```

Comment:
Could use `mapOf` as the args is used as an immutable map.

Reply:
updated